types/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import {Node} from 'unist'
+
+declare namespace sanitize {
+  /**
+   * Sanitization configuration
+   */
+  interface Schema {
+    attributes?: {
+      [key: string]: Array<string | [string, ...any[]]>
```

Comment:
HTML values can be narrowed to a set of primitives
```suggestion
      [key: string]: Array<string | [string, ...(string | number | boolean)[]]>
```